### ISSUE TITLE
How about disabling bulkhead by default?

### ISSUE THREAD

**Issue by tony612** [Shopify/semian#166]
When using semian in puma and enable bulkhead, there will be some errors raised. How about disabling bulkhead by default to avoid this problem?

**Comment by sirupsen** [comment]
Yeah, no strong opinions about this, they're hard to configure. Think I'd be OK with this. @csfrancis @byroot thoughts?

**Comment by csfrancis** [comment]
I think I'd be fine with this as well.

**Comment by miry** [comment]
Need first release a version of Gem with  warning message, that `bulkheading` is disabled by default.
